lib/middleware/request-handler.ts

```diff
@@ -455,24 +493,72 @@ export function onUnsubscribe(req: Interface.IOurRequest,
     }
 }
 
+var SUBSCRIPTION_ACTION_MAP: ActionMap = {
+    'start': onSubscribe,
+    'stop':  onUnsubscribe
+};
+
+export function getSubscriptionActions(): string[]
+{
+    return _.keys(SUBSCRIPTION_ACTION_MAP);
+}
+
 export function onChangeSubscriptions(req: Interface.IOurRequest,
                                       res: Interface.IOurResponse,
                                       next: restify.Next): void
 {
-    assert(req.apiSession, 'apiSession not found');
-    assert(req.blpSession, 'blpSession not found');
+    dispatchAction(SUBSCRIPTION_ACTION_MAP, req, res, next);
+}
+
+function onGetToken(req: Interface.IOurRequest,
```

Comment:
Is there any specific reason why we need the get token route for now? It seems like the same as an normal request to me.